spillr_paper.Rmd

```diff
@@ -279,7 +284,7 @@ We rely on assumption [(A1)](#assumption1) to justify updating only the distribu
 
 ### Spillover Decision
 
-To perform the spillover compensation, we draw from a Bernoulli distribution with the spillover probability as parameter to decide whether or not to assign a given count to spillover. We consider spillover counts as having no clear biological interpretation and mask them from our dataset while keeping all other counts. By default, we choose to set spillover counts to `NA` instead of zero to avoid zero-inflated distributions, \textcolor{red}{but the user can specify to set the spillover to any other value.}
+To perform spillover compensation, we draw from a Bernoulli distribution with the spillover probability as parameter to decide whether or not to assign a given count to spillover. \textcolor{red}{We set counts designated to spillover to a user-supplied value. We recommend a value of zero to maintain the overall cellular composition of the sample, or masking them using \texttt{NA} to retain mean values closer to the original distribution.}
```

Comment:
..or masking them using \texttt{NA} to enable downstream analyses to treat those values separately.

Reply:
I would use "marking them as \texttt{NA} or -1 to enable...", rather than the word `masking`. What you think?

Reply:
I added an example half-sentence.
Resolve if you agree.